primitives/src/main/java/org/terasology/primitives/Rectangled.java

```diff
@@ -150,6 +114,42 @@ public Rectangled(double minX, double minY, double maxX, double maxY) {
         this.maxY = maxY;
     }
 
+    @Override
+    public double minX() {
+        return this.minX;
+    }
+
+    @Override
+    public double minY() {
+        return this.minY;
+    }
+
+    @Override
+    public double maxX() {
+        return this.maxX;
+    }
+
+    @Override
+    public double maxY() {
+        return this.maxY;
+    }
+
+    @Override
+    public double getSizeX() {
+        return this.maxX - this.minX;
+    }
+
+    @Override
+    public double getSizeY() {
+        return this.maxY - this.minY;
+    }
+
+    @Override
+    public Vector2d getSize(Vector2d dest) {
+        return dest.set(this.maxX - this.minX, this.maxY - this.minY);
+    }
```

Comment:
For BlockRegion, I implemented these directly on the interface with default methods... I think in the end it doesn't matter, as the interface is only for mutable vs immutable, but not for allowing multiple implementations...